scripts/bootstrap.sh

```diff
@@ -14,10 +14,18 @@
 # limitations under the License.
 #
 
+if [[ ! -d ${CHIP_ROOT} ]]; then
```

Comment:
This seems like a mis-merge with <https://github.com/project-chip/connectedhomeip/pull/3291>?  Or at least looks like it backs out the changes from that?